analysis_scripts/analyze_benchmark_data/benchmark_analysis.md

```diff
@@ -109,66 +109,88 @@ bm_loc_file = paste(proj_dir,
-    ## 4           COUNTRY United States of America (USA)      Americas
-    ## 5 STATE_OR_PROVINCE United States of America (USA)      Americas
-    ## 6 STATE_OR_PROVINCE United States of America (USA)      Americas
-    ##   est_un_subregion
-    ## 1 Northern America
-    ## 2 Northern America
-    ## 3 Northern America
-    ## 4 Northern America
-    ## 5 Northern America
-    ## 6 Northern America
+    ##   est_country_code                                                   file_id
+    ## 1               ao us-science-academies-take-on-human-genome-editing-1.17581
+    ## 2               ar                                        d41586-020-01756-0
+    ## 3               au                                        d41586-020-00166-6
+    ## 4               au                                        d41586-020-02835-y
+    ## 5               au                                        d41586-020-02835-y
+    ## 6               au                                        d41586-020-02835-y
+    ##                                                           text          ner
+    ## 1                                                          nam ORGANIZATION
+    ## 2                                                    argentina      COUNTRY
+    ## 3                                                    australia      COUNTRY
+    ## 4                                                        csiro ORGANIZATION
+    ## 5 commonwealth scientific and industrial research organisation ORGANIZATION
+    ## 6                                        james cook university ORGANIZATION
+    ##   est_country est_un_region          est_un_subregion
+    ## 1      Angola        Africa             Middle Africa
+    ## 2   Argentina      Americas             South America
+    ## 3   Australia       Oceania Australia and New Zealand
+    ## 4   Australia       Oceania Australia and New Zealand
+    ## 5   Australia       Oceania Australia and New Zealand
+    ## 6   Australia       Oceania Australia and New Zealand
 
 Similar to before we will match columns baed on their names, in `raw_loc_df` it has `est_` columns and in `bm_loc_df` is has matching `true_` columns
 
 Now lets first look at the benchmark data ![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-1.png)![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-2.png)![](benchmark_analysis_files/figure-markdown_github/unnamed-chunk-9-3.png)
+
+Ok, so we see a strong signal that US/Americas/Europe are mentioned at a much higher rate than other regions. We would like to also see this pattern in our predicted locations, but first we need to show that our estimations are accurate. Shown below are now analyses comparing our hand-annotated benchmark data against the fully-automated processed data. We would like to show that the true number of articles with a region mention, is highly correlated to the estimated number of articles from our full pipeline.
```

Comment:
Yes, this could happen if we only consider the correlations (so the last 6 plots). However, with the confusion matrix and the somewhat strong Kappa value, this shows that this type of switching behavior is not occurring.

The reason why I focus on the number of articles per country is because this is the metric that I will use later for the analysis of the full dataset.

Does this make sense?

Reply:
Yes it does!